llm/ggml.go

```diff
@@ -81,6 +84,52 @@ func (kv KV) ContextLength() uint64 {
 	return kv.u64(fmt.Sprintf("%s.context_length", kv.Architecture()))
 }
 
+func (kv KV) ChatTemplate() string {
```

Comment:
Let's move this to a `template` package or similar to avoid increasing the surface area of `llm` which has quite a bit in it already